doc/source/ray-overview/what-is-ray.md

```diff
@@ -0,0 +1,77 @@
+# What is Ray
+
+<strong><a href="https://www.ray.io/" target="_blank">Ray</a></strong> is an open-source unified compute framework that makes it easy to scale AI and Python workloads. This guide gives a quick tour of Ray's features.
```

Comment:
@angelinalg I think you left this document in by accident, correct? All changes should be in  `index.md`.

Reply:
once you remove this doc, the CI build should also pass.

Reply:
Thanks for catching that, @maxpumperla! I'll take care of it.